components/Send/SendReview.tsx

```diff
@@ -198,6 +198,7 @@ const SendReview: React.FC = () => {
       </Button>
 
       <NeedHelp />
+      {totpModal}
```

Comment:
totpModal can be added as a new step in Send modal flow instead of directly adding it in the review step
https://github.com/Solid-Money/solid-ui/blob/d9f647cef66c574e876cbdb8e5229d271f7c53ed/constants/modals.ts#L80